orca-igor/src/main/groovy/com/netflix/spinnaker/orca/igor/tasks/GetCommitsTask.groovy

```diff
@@ -150,7 +150,8 @@ class GetCommitsTask implements DiffTask {
         clusterPair?.canary?.availabilityZones?.findResult { key, value -> key == region }
       }?.canary?.amiName
     } else if (context.deploymentDetails) { // deploy asg stage
-      return context.deploymentDetails.find { it.region == region }?.amiId
+      def details = context.deploymentDetails.find { it.region == region }
+      return details?.amiId ?: details?.ami
```

Comment:
#1043 changed `ami` to `amiId`.  If I change this to have the `ami` be first, then it will negate the change made in #1043 because `ami` is present in the `deploymentDetails` map and, in the case of the application pipelines tested, `amiId` was not.